pom.xml

```diff
@@ -64,17 +64,22 @@
     <properties>
         <revision>2.14</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.121.1</jenkins.version>
+        <jenkins.version>2.121.3</jenkins.version>
         <java.level>8</java.level>
-        <workflow-cps-plugin.version>2.69</workflow-cps-plugin.version>
-        <git-plugin.version>3.6.4</git-plugin.version>
-        <scm-api-plugin.version>2.2.7</scm-api-plugin.version>
-        <workflow-scm-step-plugin.version>2.4</workflow-scm-step-plugin.version>
-        <workflow-multibranch-plugin.version>2.10</workflow-multibranch-plugin.version>
-        <workflow-step-api-plugin.version>2.19</workflow-step-api-plugin.version>
+        <workflow-cps-plugin.version>2.71-rc2340.ea3a8e1ac84d</workflow-cps-plugin.version>
```

Comment:
```suggestion
        <workflow-cps-plugin.version>2.71-rc2340.ea3a8e1ac84d</workflow-cps-plugin.version> <!-- TODO https://github.com/jenkinsci/workflow-cps-plugin/pull/297 -->
```

Reply:
Yeah, I need to do better about remembering TODO comments for this kind of thing. I updated to the released version in d26fa6c.